### PR TITLE
feat: update ixc2 model to handle video chunks

### PR DESCRIPTION
**Pull request by CamiloInx**
Description
---
* Allow IXC2 model to handle video chunk analysis.

### CODE REVIEW

File: vision_agent_tools/models/internlm_xcomposer2.py [hunk 1]
```diff
@@ -76,7 +76,8 @@ def __call__(
         image: Image.Image | None = None,
         video: VideoNumpy | None = None,
         frames: Frames = MAX_NUMBER_OF_FRAMES,
-    ) -> str:
+        chunk_size: int = 1,
```
Comment (camiloaz) [inline vision_agent_tools/models/internlm_xcomposer2.py:79]: let's call it `n_chunks` instead, `chunk size` could be confusing.

**Review by camiloaz — COMMENTED** [review]
Do we have tests for this? If so, it should be failing for the image case because now the response is a list with a single item. Can we also add a test for chunks > 1?

**Comment by CamiloInx** [comment]
> Do we have tests for this? If so, it should be failing for the image case because now the response is a list with a single item. Can we also add a test for chunks > 1?

@camiloaz No, the models don't have tests incorporated yet on vision-agent-tools. We do tests it on the other repositories as integration tests.

File: vision_agent_tools/models/internlm_xcomposer2.py (hunk 1 above)
Comment (CamiloInx) [reply vision_agent_tools/models/internlm_xcomposer2.py]: Done! I changed the name

**Comment by CamiloInx** [comment]
> > > Do we have tests for this? If so, it should be failing for the image case because now the response is a list with a single item. Can we also add a test for chunks > 1?
> > 
> > 
> > @camiloaz No, the models don't have tests incorporated yet on vision-agent-tools. We do tests it on the other repositories as integration tests.
> 
> We do have tests:
> 
> https://github.com/landing-ai/vision-agent-tools/blob/4f856bd991009616ef2e4c55e9a41872b28f2ec5/tests/tools/test_internlm_xcomposer2.py#L5-L25

Yeah, sorry for that. I forgot that we moved folder where we used to store the tests. Added the corresponding unit test.

File: tests/models/test_internlm_xcomposer2.py [hunk 2]
```diff
@@ -13,6 +13,17 @@ def test_successful_internlm_xcomposer2_for_video(random_video_generator):
...
 
+def test_successful_internlm_xcomposer2_for_video_chunks(random_video_generator):
+    video_np = random_video_generator(n_frames=10)
+    prompt = "Here are some frames of a video. Describe this video in detail"
+
+    run_inference = InternLMXComposer2()
+
+    answer = run_inference(video=video_np, prompt=prompt, n_chunks=2)
+
+    assert len(answer) > 1
```
Comment (camiloaz) [inline tests/models/test_internlm_xcomposer2.py:24]: This should be == 2, right?

**Review by camiloaz — APPROVED** [review]

File: tests/models/test_internlm_xcomposer2.py (hunk 2 above)
Comment (CamiloInx) [reply tests/models/test_internlm_xcomposer2.py:24]: Yes, let me fix it

**Pull request merged**